Libraries/LibWeb/TrustedTypes/TrustedHTML.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2025, Miguel Sacristán Izcue <miguel_tete17@hotmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/TrustedTypes/TrustedHTML.h>
+
+#include <LibGC/Ptr.h>
+#include <LibJS/Runtime/Realm.h>
+#include <LibWeb/Bindings/Intrinsics.h>
+
+namespace Web::TrustedTypes {
+
+GC_DEFINE_ALLOCATOR(TrustedHTML);
+
+TrustedHTML::TrustedHTML(JS::Realm& realm, Utf16String data)
+    : PlatformObject(realm)
+    , m_data(move(data))
+{
+}
+
+void TrustedHTML::initialize(JS::Realm& realm)
+{
+    WEB_SET_PROTOTYPE_FOR_INTERFACE(TrustedHTML);
+    Base::initialize(realm);
+}
+
+// https://w3c.github.io/trusted-types/dist/spec/#trustedhtml-stringification-behavior
+Utf16String const& TrustedHTML::to_string() const
+{
+    // 1. return the associated data value.
```

Comment:
nit: We don't typically add a number to spec text that isn't numbered like this. I think most of the time for simple getters like this we just omit the text entirely.

I think the same comment applies to all `to_string()` and `to_json()` methods that currently follow this pattern.

Reply:
It was so initially but I was asked by @Lubrsi to change it.

I am cool with both options :)

Reply:
Fair enough, I see the comment now. I don't feel too strongly about it.